src/translation/translation_nb_NO.ts

```diff
@@ -736,17 +736,17 @@
     <message>
         <location filename="../clientdlg.cpp" line="286"/>
         <source>&amp;Connection Setup...</source>
-        <translation>&amp;Tilkoblingsoppsett …</translation>
+        <translation>&amp;Tilkoblingsoppsett...</translation>
     </message>
     <message>
         <location filename="../clientdlg.cpp" line="290"/>
         <source>&amp;Load Mixer Channels Setup...</source>
-        <translation type="unfinished">&amp;Last inn kanalmikserinnstillingene …</translation>
```

Comment:
I believe the initial ... was typographically better.

Reply:
UK keyboards do not have that character.  I've no idea what UTF-8 character it is.  Ideally, the character would be coded as `&whatever;` so it was easy to see (like `&amp;`).

Reply:
In Jamulus, both English and all the other translations use a plain three dots `...`, so I changed nb_NO to the same for consistency.

Also, in the font on my RPi, at least as viewed via VNC, the ellipsis character was very indistinct, looking just like a small underscore.

Just for reference, in reply to @pljones, the ellipsis is at position 0x85 in Windows-1252 encoding, can be input on a PC using Alt-0133, and in UTF-8 is code point `0x2026`. In HTML, it is `&hellip;` (for "horizontal ellipsis): &hellip;